watershed_workflow/hydrography.py

```diff
@@ -473,12 +472,25 @@ def prune_river_by_fractional_contributing_area(river, area_fraction, total_area
     if total_area is None:
         total_area = river.properties['TotalDrainageAreaSqKm']
     logging.info(f'... total contributing area = {total_area}')
-        
+
+    from shapely.ops import unary_union
```

Comment:
I would rather you imported shapely.ops at the top of the file and then used shapely.ops.unary_union rather than importing specific names.